Scripts/Biochemistry/Add_New_Reactions.py

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/env python
+import os, sys, re, copy
+from csv import DictReader
+from collections import OrderedDict
+temp=list();
+header=1;
+
+Biochem="KEGG"
+Biochem_Root="../../Biochemistry/Aliases/Provenance/Primary_Databases/";
+
+sys.path.append('../../Libs/Python')
+from BiochemPy import Reactions, Compounds, InChIs
+
+CompoundsHelper = Compounds()
+Compounds_Dict = CompoundsHelper.loadCompounds()
+ReactionsHelper = Reactions()
+Reactions_Dict = ReactionsHelper.loadReactions()
+Reactions_Codes = ReactionsHelper.generateCodes(Reactions_Dict)
+
+Default_Rxn = {"id":"cpd00001","name":"null","abbreviation":"null","aliases":"null",
+               "code":"null","stoichiometry":"null","equation":"null","definition":"null",
+               "reversibility":"=","direction":"=","deltag":"10000000","deltagerr":"10000000",
+               "status":"NB","is_obsolete":0,"is_transport":0,
+               "abstract_reaction":"null","pathways":"null","ec_numbers":"null",
+               "compound_ids":"null","linked_reaction":"null","notes":"null","source":""}
+
+Compounds_Alias_Dict=CompoundsHelper.loadMSAliases()
+Source_Alias_Dict = dict()
+for msid in Compounds_Alias_Dict.keys():
+    for source in Compounds_Alias_Dict[msid].keys():
+        if(source not in Source_Alias_Dict):
+            Source_Alias_Dict[source]=dict()
+        for alias in Compounds_Alias_Dict[msid][source]:
+            if(alias not in Source_Alias_Dict[source]):
+                Source_Alias_Dict[source][alias]=list()
+            Source_Alias_Dict[source][alias].append(msid)
+
+Original_Alias_Dict=ReactionsHelper.loadMSAliases()
+New_Alias_Count=dict()
+
+Names_Dict = ReactionsHelper.loadNames()
+All_Names = dict()
+New_Name_Count=dict()
+for msid in sorted(Names_Dict.keys()):
+    for name in Names_Dict[msid]:
+        All_Names[name]=1
```

Comment:
Looks like you want the Counter type here: https://docs.python.org/3.6/library/collections.html#collections.Counter

You can also iterate over values in a dict directly using `.values()`
Together this would look like:
```suggestion
All_Names = Counter()
for names in Names_Dict.values():
    All_names.update(names)
```

Reply:
Looks like I can't do multi line suggestions on github...